athena/templates/tdr_cloudtrail_logs_mgmt.sql.tpl

```diff
@@ -0,0 +1,8 @@
+SELECT *
+FROM ${database_name}."tdr_cloudtrail_logs_mgmt"
+WHERE
+  eventsource = 'lambda.amazonaws.com' AND
+  eventtime >= '2020-05-15' AND
```

Comment:
Are the dates hard-coded so that we can use this query as an example when we're running real queries?

Reply:
yes, these queries are intended as examples